### ISSUE TITLE
Problem installing the docker for the backend

### ISSUE THREAD

**Issue by javiquinte** [iris-edu/yasmine-stationxml-editor#30]
Trying to create the docker container I got the following error

```
[...]
 => [ 6/11] RUN apt-get update                                                                                                                           2.0s
 => ERROR [ 7/11] RUN apt-get -y install google-chrome-stable                                                                                            1.4s
------
 > [ 7/11] RUN apt-get -y install google-chrome-stable:
#10 0.324 Reading package lists...
#10 1.076 Building dependency tree...
#10 1.200 Reading state information...
#10 1.272 Some packages could not be installed. This may mean that you have
#10 1.272 requested an impossible situation or if you are using the unstable
#10 1.272 distribution that some required packages have not yet been created
#10 1.272 or been moved out of Incoming.
#10 1.272 The following information may help to resolve the situation:
#10 1.272
#10 1.272 The following packages have unmet dependencies:
#10 1.340  google-chrome-stable : Depends: libdrm2 (>= 2.4.75) but 2.4.74-1 is to be installed
#10 1.340                         Depends: libgbm1 (>= 17.1.0~rc2) but 13.0.6-1+b2 is to be installed
#10 1.340                         Depends: libu2f-udev but it is not installable
#10 1.365 E: Unable to correct problems, you have held broken packages.
------
executor failed running [/bin/sh -c apt-get -y install google-chrome-stable]: exit code: 100
```

**Comment by javiquinte** [comment]
Sorry! This is a duplicate of the issue #27

**Comment by rcasey-earthscope** [comment]
I think that this may have been solved with #33.
